helfomat-docker/docker-compose.yml

```diff
@@ -32,35 +32,14 @@ services:
     - 9200:9200
     - 9300:9300
 
-  zoo1:
-    image: zookeeper:3.4.9
-    container_name: helfomat-zookeeper1
-    hostname: zoo1
-    ports:
-    - "2181:2181"
+  postgres:
+    image: postgres:10
+    container_name: helfomat-postgres
     environment:
-      ZOO_MY_ID: 1
-      ZOO_PORT: 2181
-      ZOO_SERVERS: server.1=zoo1:2888:3888
+      POSTGRES_USER: helfenkannjeder
+      POSTGRES_PASSWORD: helfenkannjeder
+      POSTGRES_DB: helfenkannjeder
     volumes:
-    - ./data/zoo1/data:/data
-    - ./data/zoo1/datalog:/datalog
-
-  kafka1:
-    image: confluentinc/cp-kafka:5.0.0
-    hostname: kafka1
-    container_name: helfomat-kafka1
+    - ./data/postgres:/var/lib/postgresql/data
```

Comment:
Create a volume in the compose and reference it here. This kind of directory mounting does not work well when you have a windows system.

Add this to the end of the compose file:
```
volumes:
  postgres-db-data:
```

And mount it like this:
```
postgres-db-data:/var/lib/postgresql/data/
```

Reply:
I changed it. I used the other syntax only for compatibility reasons with elasticsearch.